hms-app/src/components/card-details/HackathonDetails.tsx

```diff
@@ -261,7 +262,7 @@ export default function HackathonDetails(props: IProps) {
         </div>
       )}
 
-      {hackathonData.startDate &&
+      {hackathonData.startDate != NULL_DATE &&
```

Comment:
```suggestion
      {hackathonData.startDate !== NULL_DATE &&
```

Reply:
and the same thing in the other places too

Reply:
updated